shared/src/vyxal/Elements.scala

```diff
@@ -1002,6 +1002,19 @@ object Elements:
           ctx.push(s.dropRight(1), if s.isEmpty then "" else s.last.toString)
         case arg => throw UnimplementedOverloadException("ṫ", List(arg))
     },
+    addFull(
+      Monad,
+      "øh",
```

Comment:
Why is this in the string digraph namespace? I would have thought it'd be better suited for the list digraph namespace (`Þ`)

Reply:
no idea tbh

Reply:
```suggestion
      "Þh",
```
Then you can call it `Þh`